irodsadmin/tests/test_commands.py

```diff
@@ -206,49 +209,59 @@ def test_is_assay_or_study_invalid(self):
 
     def test_get_orphans_none(self):
         """Test get_orphans() with no orphans available"""
-        self.assertListEqual(
-            irodsorphans.get_orphans(
-                self.irods,
-                self.irods_backend,
-                self.expected_collections,
-                [self.assay],
-            ),
-            [],
+        # Capture stdout
+        out = io.StringIO()
+        sys.stdout = out
+        irodsorphans.get_orphans(
+            self.irods,
+            self.irods_backend,
+            self.expected_collections,
+            [self.assay],
         )
+        output = out.getvalue()
+        sys.stdout = sys.__stdout__
+
+        self.assertEqual('', output)
 
     def test_get_orphans_assay(self):
         """Test get_orphans() with orphan assay"""
         orphan_path = '{}/assay_{}'.format(
             self.irods_backend.get_path(self.study), str(uuid.uuid4())
         )
         self.irods.collections.create(orphan_path)
-        self.assertListEqual(
-            irodsorphans.get_orphans(
-                self.irods,
-                self.irods_backend,
-                self.expected_collections,
-                [self.assay],
-            ),
-            [orphan_path],
+        # Capture stdout
+        out = io.StringIO()
+        sys.stdout = out
+        irodsorphans.get_orphans(
+            self.irods,
+            self.irods_backend,
+            self.expected_collections,
+            [self.assay],
         )
+        output = out.getvalue()
+        sys.stdout = sys.__stdout__
+        self.assertIn(orphan_path, output)
 
     def test_get_orphans_study(self):
         """Test get_orphans() with orphan study"""
         orphan_path = '{}/sample_data/study_{}'.format(
             self.irods_backend.get_path(self.project), str(uuid.uuid4())
         )
         self.irods.collections.create(orphan_path)
-        self.assertListEqual(
-            irodsorphans.get_orphans(
-                self.irods,
-                self.irods_backend,
-                self.expected_collections,
-                [self.assay],
-            ),
-            [orphan_path],
+        # Capture stdout
+        out = io.StringIO()
+        sys.stdout = out
+        irodsorphans.get_orphans(
+            self.irods,
+            self.irods_backend,
+            self.expected_collections,
+            [self.assay],
         )
+        output = out.getvalue()
+        sys.stdout = sys.__stdout__
+        self.assertIn(orphan_path, output)
```

Comment:
Here is `assertIn` and not `assertEqual` due to some additional information in stdout. Here is an output:
```
    self.assertEqual(orphan_path, output)
AssertionError: '/sodarZone/projects/fa/fa2a0714-7c97-44b9[70 chars]3456' != 'fa2a0714-7c97-44b9-b762-1f370cac6828;Test[131 chars]es\n'
- /sodarZone/projects/fa/fa2a0714-7c97-44b9-b762-1f370cac6828/landing_zones/user/small_germline_study/20201031_123456
+ fa2a0714-7c97-44b9-b762-1f370cac6828;TestProject;/sodarZone/projects/fa/fa2a0714-7c97-44b9-b762-1f370cac6828/landing_zones/user/small_germline_study/20201031_123456;0;0 bytes
```